scss/components/side-nav.scss

```diff
@@ -75,7 +75,7 @@ $block: #{$fd-namespace}-side-nav;
     &__link,
     &__sublink {
         display: flex;
-        align-items: center;
+        align-items: flex-start;
```

Comment:
It seems like there's a balance that can be achieved here.  This keeps the icon anchored to the top, however, for single line text, the icon doesn't align to it very well (it's almost the same as the original state, just off on the bottom side).  I think the desired effect is to have the icon appear to be center-aligned to the first row of text regardless of whether there is one or two lines of text.
![Screen Shot 2019-06-19 at 10 23 29 AM](https://user-images.githubusercontent.com/11407353/59779147-5b30c180-927d-11e9-960f-9f2871bb3445.png)